src/software/backend/replay_logging/replay_logger.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <experimental/filesystem>
+
+#include "software/multithreading/last_in_first_out_threaded_observer.h"
+#include "software/proto/replay_msg.pb.h"
+#include "software/proto/sensor_msg.pb.h"
+
+
+class ReplayLogger : public LastInFirstOutThreadedObserver<SensorMsg>
+{
+   public:
+    /**
+     * Constructs a Replay Logger. _msgs_per_chunk is a parameter that sets
+     * how many individual SensorMsg's go into one replay_logging 'chunk' file.
+     * We separate replays into files of a certain number of messages to
+     * reduce the amount of lost data in the case of a crash.
+     * @param output_directory The directory that we output replay_logging chunk files to.
```

Comment:
Is this directoy path global? Local to the cwd when we run the program?

Reply:
it should be the absolute path, added to jdoc comment